src/silverlabnwb/nwb_file.py

```diff
@@ -943,7 +980,8 @@ def add_rois(self, roi_path):
         self.log('Loading ROI locations from {}', roi_path)
         assert os.path.isfile(roi_path)
         roi_data = pd.read_csv(
-            roi_path, sep='\t', header=0, index_col=False, dtype=np.float16, memory_map=True)
+            roi_path, sep='\t', header=0, index_col=False, dtype=np.float16,
+            converters={'Z start': np.float64, 'Z stop': np.float64}, memory_map=True)
```

Comment:
We now know that some of these can be read as integers (#59), but I suggest leaving it like this for now.